File: filedata/entity/c#/Websiteinfomation.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Coldairarrow.Entity.Video
{
    /// <summary>
    /// 
    /// </summary>
    [Table("website_infomation")]
    public class Websiteinfomation
    {
        
        /// <summary>
        /// 站点名称 主键
        /// </summary>
        [Column("CompanyStyle")]
        public string CompanyStyle { get; set; }
    
    
        /// <summary>
        /// 体验卡价格 1天
        /// </summary>
        [Column("DayCardPrice")]
        public decimal DayCardPrice { get; set; }
    
    
        /// <summary>
        /// 周卡价格 7天
        /// </summary>
        [Column("WeekCardPrice")]
        public decimal WeekCardPrice { get; set; }
    
    
        /// <summary>
        /// 月卡价格 30天
        /// </summary>
        [Column("MonthCardPrice")]
        public decimal MonthCardPrice { get; set; }
    
    
        /// <summary>
        /// 季卡价格 90天
        /// </summary>
        [Column("QuarterlyCardPrice")]
        public decimal QuarterlyCardPrice { get; set; }
    
    
        /// <summary>
        /// 年卡价格 360天
        /// </summary>
        [Column("YearCardPrice")]
        public decimal YearCardPrice { get; set; }
    
    
        /// <summary>
        /// 创建时间
        /// </summary>
        [Column("CreateTime")]
        public DateTime CreateTime { get; set; }
    
    
    }
}

File: filedata/entity/java/websiteInfomation.java

package entity;
// 
public class Websiteinfomation  extends BaseDomain{
    public Websiteinfomation() {

    }
    
    //
    private String CompanyStyle;
        
    //
    private double DayCardPrice;
        
    //
    private double WeekCardPrice;
        
    //
    private double MonthCardPrice;
        
    //
    private double QuarterlyCardPrice;
        
    //
    private double YearCardPrice;
        
    //
    private Date CreateTime;
        
    public String getCompanystyle() {
        return Companystyle;
    }
    public void setCompanystyle(String Companystyle) {
        this.Companystyle = Companystyle;
    }
            
    public double getDaycardprice() {
        return Daycardprice;
    }
    public void setDaycardprice(double Daycardprice) {
        this.Daycardprice = Daycardprice;
    }
            
    public double getWeekcardprice() {
        return Weekcardprice;
    }
    public void setWeekcardprice(double Weekcardprice) {
        this.Weekcardprice = Weekcardprice;
    }
            
    public double getMonthcardprice() {
        return Monthcardprice;
    }
    public void setMonthcardprice(double Monthcardprice) {
        this.Monthcardprice = Monthcardprice;
    }
            
    public double getQuarterlycardprice() {
        return Quarterlycardprice;
    }
    public void setQuarterlycardprice(double Quarterlycardprice) {
        this.Quarterlycardprice = Quarterlycardprice;
    }
            
    public double getYearcardprice() {
        return Yearcardprice;
    }
    public void setYearcardprice(double Yearcardprice) {
        this.Yearcardprice = Yearcardprice;
    }
            
    public Date getCreatetime() {
        return Createtime;
    }
    public void setCreatetime(Date Createtime) {
        this.Createtime = Createtime;
    }
             
    @Override
    public String toString() {
        return "website_infomation[CompanyStyle=" + CompanyStyle + ",DayCardPrice=" + DayCardPrice + ",WeekCardPrice=" + WeekCardPrice + ",MonthCardPrice=" + MonthCardPrice + ",QuarterlyCardPrice=" + QuarterlyCardPrice + ",YearCardPrice=" + YearCardPrice + ",CreateTime=" + CreateTime + "]";
    }
        
    
}
        